### ISSUE TITLE
function term for "tRNA wobble position uridine thiolation"

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#13257]
http://www.ebi.ac.uk/QuickGO/GTerm?id=GO:0002143

There is no MF term for this activity, presumably because it is performed by multiple gene products (but is this a reason not to have an MF term?)

In E. coli, the first step of the reaction is reductive elimination of sulfur from L-cysteine by IscS cysteine desulfurase to form an enzyme-bound cysteine-persulfide intermediate. Then, five essential gene products, TusA, TusB, TusC, TusD and TusE, mediate a sulfur relay that delivers the terminal sulfur of persulfide from IscS to MnmA12. The last protein, MnmA catalyzes the transfer of the sulfur from IscS to an ATP activated U34 of the tRNA.

So which MF term would we use?  This seems similar to ubiquitination, and we have MF terms for these.....Presumably we should have MF terms for this pathway as its fairly well characterized....so its a shame to only have BP...

**Comment by hdrabkin** [comment]
If the thiolation requires multiple steps, then the activities should be the mfs, but the term above would have to be a process. However, when I get back, I'll make sure there isn't a one-step activity out there.

**Comment by hdrabkin** [comment]
I can't find any evidence for a single step activity that does this for wobble base thiol modifications.
  This is because the thiolated products in the anticodon are complex: eg, mnm5s2U, mcm5s2U. Not like at position 8 or 9, for s4U, which is carried out by a single gene product. 
See for example Table 1 in "Biosynthesis and Function of Modified Nucleosides (GR Bjork), in tRNA: Structure, Biosynthesis, and Function (eds: Soll and RajBhandar).

**Comment by ValWood** [comment]
So which MF term describes this activity?

**Comment by hdrabkin** [comment]
Will need some time to dig this out.
https://www.researchgate.net/figure/51610626_fig5_FIG-6-Uridine-modification-by-MnmE-MnmA-in-collaboration-with-other-proteins-carries
I seem to have lost my ability to attach a file here
see url
sothe steps involved in one type of final modification; the first step is S2U34, FIG. 6. Uridine modification by MnmE. MnmA (in collaboration with other proteins) carries out thiolation at the 2-position of the wobble uridine, whereas an ␣ 2 ␤ 2 heterotetrameric complex formed by MnmE and MnmG independently catalyzes the first step of the cmnm 5 modification at the 5-position. MnmC has two enzymatic activities that transform the cmnm 5 intermediate into the final mnm 5 modification in certain bacteria. (Adapted from reference 283 by permission of Oxford University Press.)

**Comment by hdrabkin** [comment]
Now that I know the gene or a gene,found
a uridine34 in tRNA + a [TusE sulfur carrier protein]-S-sulfanylcysteine + ATP + a reduced electron acceptor → a 2-thiouridine34 in tRNA + a [TusE sulfur carrier protein]-L-cysteine + AMP + an oxidized electron acceptor + diphosphate + H+
If I don't find something in ec or metacyc, I'll need to create the term

**Comment by hdrabkin** [comment]
From HumanCyc
TusE functions as a sulfur mediator during synthesis of 2-thiouridine of the modified wobble base mnm5s2U in tRNA. TusE directly interacts with both the TusBCD complex and the MnmA-tRNA complex, but not with tRNA itself [Ikeuchi06].
This is getting messy.

2-thiouridine formation in vitro is greatly enhanced in the presence of TusE. TusE is labeled by 35S sulfur during 2-thiouridine formation in vitro [Ikeuchi06].
A tusE deletion mutant lacks mnm5s2U, accumulates mnm5U and has a growth defect. The cysteine residue C108 is required for sulfur transfer [Ikeuchi06].

**Comment by hdrabkin** [comment]
NTR: tRNA U34 thiolation activity
def: Catalysis of the reaction:  uridine34 in tRNA + a [TusE sulfur carrier protein]-S-sulfanylcysteine + ATP + a reduced electron acceptor → a 2-thiouridine34 in tRNA + a [TusE sulfur carrier protein]-L-cysteine + AMP + an oxidized electron acceptor + diphosphate + H+
Related Synonym: tRNA-specific 2-thiouridylase
Exact Synonym: tRNA U34 2-thiouridylase
PMID:16387657 
PMID:12549933
is_a various terms already existing;

**Comment by ValWood** [comment]
Hi Harold,

It seems that this term just needs to be added?

Val

**Comment by hdrabkin** [comment]
I was going to make an ec request first, but THEN I read that this enzyme appears to be the last step, but I need to confirm this (that is, you don't find a U34thio-S by itself. ) 
Will sort out in a bit.

**Comment by hdrabkin** [comment]
Ok, I thought I read something different, but PMID:12549933 has a figure that outlines the steps. So I'm putting this in now.
![screen shot 2017-09-05 at 2 58 51 pm](https://user-images.githubusercontent.com/10440080/30078411-0f7324c4-924b-11e7-9759-87d9d27bae33.jpg)

**Comment by hdrabkin** [comment]
Done. GO:0106054, tRNA U34 thiol-transferase activity

**Comment by ValWood** [comment]
hmm, is this the same thing?
https://www.yeastgenome.org/go/GO:0061708
at sgd this is
tRNA-specific 2-thiouridylase; responsible for 2-thiolation of the wobble base of mitochondrial tRNAs

and has
Gene Ontology Term: tRNA-5-taurinomethyluridine 2-sulfurtransferase
GO ID
GO:0061708
Aspect
Molecular Function
Description
Catalysis of 5-taurinomethyluridine in tRNA + a [protein]-S-sulfanylcysteine + ATP + a reduced electron acceptor = a 5-taurinomethyl-2-thiouridine in tRNA + a [protein]-L-cysteine + AMP + an oxidized electron acceptor + diphosphate + H+.

is taurinomethyluridine an alternative name?

**Comment by ValWood** [comment]
not to self, I think I'm looking at ctu1/2 cyto and slm3 mito

**Comment by mah11** [comment]
Not that I know sfa about this stuff, but none of the structures in Harold's figure match the structure CHEBI has for 5-taurinomethyluridine

http://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:87172

**Comment by ValWood** [comment]
Hmm, I'm either looking at the wrong thing, or this is multifunctional....or is it some intermediate...

Harold, any ideas?

**Comment by hdrabkin** [comment]
Well, I was just looking to make sure the thiolation occurred first. 
The Def came from the gene entry for   MmnA that I saw on Metacyc (but didn't have a metacyc id). 
5-taurinomethyluridine looks like a sufate derivative. 
Note that the reaction I just added uses cysteine as a donor, 

The product of the reaction that Val found is CHEBI:87171,  has BOTH a sulfate group (at 5) AND a thiol group at 2. Note the same reaction.
